apps/src/signUpFlow/FinishTeacherAccount.tsx

```diff
@@ -0,0 +1,109 @@
+import React, {useState} from 'react';
+
+import {Button, buttonColors} from '@cdo/apps/componentLibrary/button';
+import {
+  Heading2,
+  BodyTwoText,
+  BodyThreeText,
+} from '@cdo/apps/componentLibrary/typography';
+import locale from '@cdo/apps/signup/locale';
+import SchoolDataInputs from '@cdo/apps/templates/SchoolDataInputs';
+
+import Checkbox from '../componentLibrary/checkbox/Checkbox';
+
+import style from './finish-teacher-account.module.scss';
+
+const USER_NAME_SESSION_KEY = 'username';
```

Comment:
tiny tiny nit: should this be `userName` as "user" and "name" are separated by underscores?

This is _definitely_ not worth changing if nothing else needs to change 😅